mage_ai/frontend/components/ActionForm/actions.ts

```diff
@@ -25,6 +25,7 @@ const columns: {
       Add @transformer_action, then begin typing Python code inside a function. \
       You may refer to the current dataset as df.',
     code: {
+      default: CODE_EXAMPLE,
```

Comment:
if this `custom` property is the exact same in columns and rows constants, let's create a separate variable for it and  use it in both places. It's a hassle to have to update the code in two places every time we make a change.

Reply:
That's what code example is for, or am i misunderstanding what you mean?

Reply:
meant the whole `custom` prop: 
```
custom: {
    title: 'Custom code',
    description: 'Write your own custom cleaning function. \
      Add @transformer_action, then begin typing Python code inside a function. \
      You may refer to the current dataset as df.',
    code: {
      default: CODE_EXAMPLE,
      multiline: true,
      values: USER_INPUT,
    },
```

Reply:
One is for the dataset page and the other is for column specific page

Reply:
Should I write different copy for them then?

Reply:
This is the logic that selects which to look at: (axis === AxisEnum.ROW ? actions.rows : actions.columns)?.[actionType];
Adding a third variable might break it.